app/models/page_sequence.rb

```diff
@@ -17,6 +17,7 @@ class PageSequence
     "leadership-position",
     "mostly-performed-leadership-duties",
     "eligibility-confirmed",
+    "information-provided",
```

Comment:
Seems strange to me to tell people about how we'll use their eligibility data _after_ we've asked them for it.